modular_skyrat/modules/modular_implants/code/soulcatcher/handheld_soulcatcher.dm

```diff
@@ -10,18 +10,24 @@
 	lefthand_file = 'icons/mob/inhands/items/devices_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/items/devices_righthand.dmi'
 	slot_flags = ITEM_SLOT_BELT
+	obj_flags = UNIQUE_RENAME
```

Comment:
Does that mention it in the desc that you can use a pen on it?

Reply:
Yeah, the unique rename flag automatically adds to the examine of an item if it can be renamed.

Reply:
Alright, then I'm all good with this.